src/entities/database.js

```diff
@@ -284,6 +284,43 @@ class DB implements Database {
     )
   }
 
+  async queryLoosePendingTxs() {
```

Comment:
The term "loose pending txs" means - those txs that are not in a snapshot yet. E.g. txs that were received after committing the last snapshot. Here we are returning the whole "pending set", which is  the union of `last pending snapshot` + `loose pending txs`.

Let's rename the function into `queryPendingSet` to make it correct.